src/stuf/stuf_bg/templates/stuf_bg/tests/responses/StufBgResponse.xml

```diff
@@ -20,9 +20,10 @@
                 </StUF:ontvanger>
                 <StUF:referentienummer>{{ referentienummer }}</StUF:referentienummer>
                 <StUF:tijdstipBericht>{{ tijdstip_bericht }}</StUF:tijdstipBericht>
+                <StUF:entiteittype>NPS</StUF:entiteittype>
             </ns:stuurgegevens>
             <ns:parameters>
-                <StUF:indicatorAfnemerIndicatie>false</StUF:indicatorAfnemerIndicatie>
+                <StUF:indicatorVervolgvraag>false</StUF:indicatorVervolgvraag>
```

Comment:
I can't judge the correctness of this and also don't see more details in your commit message. Did you get this from the XSDs?

Reply:
From stuf0301.xsd
```xml
	<complexType name="ParametersAntwoordSynchroon">
		<annotation>
			<documentation>Type voor gebruik in La01, La07, La09, La11 en La13 berichten.</documentation>
		</annotation>
		<complexContent>
			<restriction base="StUF:ParametersAntwoordSynchroonFormeel">
				<sequence>
					<element name="indicatorVervolgvraag" type="boolean"/>
					<element name="indicatorAfnemerIndicatie" type="boolean" default="false" minOccurs="0"/>
					<element name="aantalVoorkomens" type="StUF:AantalVoorkomens" minOccurs="0"/>
				</sequence>
			</restriction>
		</complexContent>
	</complexType>
```